scripts/install.sh

```diff
@@ -0,0 +1,177 @@
+    echo "[ERROR] Example: INVENIO_POSTGRESQL_DBPASS=dbpass123"
+    exit 1
+fi
+if [ -v ${INVENIO_REDIS_HOST} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_REDIS_HOST before runnning this script."
+    echo "[ERROR] Example: export INVENIO_REDIS_HOST=192.168.50.12"
+    exit 1
+fi
+if [ -v ${INVENIO_ELASTICSEARCH_HOST} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_ELASTICSEARCH_HOST before runnning this script."
+    echo "[ERROR] Example: export INVENIO_ELASTICSEARCH_HOST=192.168.50.13"
+    exit 1
+fi
+if [ -v ${INVENIO_RABBITMQ_HOST} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_RABBITMQ_HOST before runnning this script."
+    echo "[ERROR] Example: export INVENIO_RABBITMQ_HOST=192.168.50.14"
+    exit 1
+fi
+if [ -v ${INVENIO_WORKER_HOST} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_WORKER_HOST before runnning this script."
+    echo "[ERROR] Example: export INVENIO_WORKER_HOST=192.168.50.15"
+    exit 1
+fi
+
+# load virtualenvrapper:
+source $(which virtualenvwrapper.sh)
+
+# sphinxdoc-create-virtual-environment-begin
+mkvirtualenv ${INVENIO_WEB_VENV}
+cdvirtualenv
+mkdir -p src
+cd src
+# sphinxdoc-create-virtual-environment-end
+
+# quit on errors and unbound symbols:
+set -o errexit
+set -o nounset
+
+# sphinxdoc-install-invenio-full-begin
+# FIXME we pin older version that works:
+pip install invenio-records==v1.0.0a3
+# FIXME we need to install PostgreSQL DB connect option explicitly:
+pip install invenio-db[postgresql]
+# now we can install full Invenio:
+pip install invenio[full] --pre
+# sphinxdoc-install-invenio-full-end
+
+# sphinxdoc-create-instance-begin
+inveniomanage instance create ${INVENIO_WEB_INSTANCE}
+# sphinxdoc-create-instance-end
+
+# sphinxdoc-customise-instance-begin
+mkdir -p ../var/${INVENIO_WEB_INSTANCE}-instance/
+echo "# Database" > ../var/${INVENIO_WEB_INSTANCE}-instance/${INVENIO_WEB_INSTANCE}.cfg
+echo "SQLALCHEMY_DATABASE_URI='postgresql+psycopg2://${INVENIO_POSTGRESQL_DBUSER}:${INVENIO_POSTGRESQL_DBPASS}@${INVENIO_POSTGRESQL_HOST}:5432/${INVENIO_POSTGRESQL_DBNAME}'" >> ../var/${INVENIO_WEB_INSTANCE}-instance/${INVENIO_WEB_INSTANCE}.cfg
+echo "" >> ../var/${INVENIO_WEB_INSTANCE}-instance/${INVENIO_WEB_INSTANCE}.cfg
+echo "# Redis" >> ../var/${INVENIO_WEB_INSTANCE}-instance/${INVENIO_WEB_INSTANCE}.cfg
+echo "CACHE_REDIS_HOST='${INVENIO_REDIS_HOST}'" >> ../var/${INVENIO_WEB_INSTANCE}-instance/${INVENIO_WEB_INSTANCE}.cfg
+echo "BROKER_URL='redis://${INVENIO_REDIS_HOST}:6379/0'" >> ../var/${INVENIO_WEB_INSTANCE}-instance/${INVENIO_WEB_INSTANCE}.cfg
+echo "CELERY_RESULT_BACKEND='redis://${INVENIO_REDIS_HOST}:6379/1'" >> ../var/${INVENIO_WEB_INSTANCE}-instance/${INVENIO_WEB_INSTANCE}.cfg
+# sphinxdoc-customise-instance-end
+
+# sphinxdoc-run-bower-begin
+cd ${INVENIO_WEB_INSTANCE}
+pip install ipaddr # FIXME this seems explicitly needed on Python-2.7
+python manage.py bower
```

Comment:
Yes, but all this can come as a later improvement after the branches are merged and packages are released on PyPI.  E.g. pidstore also has to be merged and released for the Record UUID package to work; currently it does not.  E.g. bower will be killed soon, etc.  These are continual improvements that will keep coming. 

The current kickstart set of scripts work for the currently released packages on PyPI, and thus provide a base for people to easily test (and amend) installation recipes in case new packages are released on PyPI.  (Example: the move from less to sass that happened yesterday, I already incorporated this in this PR.)